app/mailers/application_mailer.rb

```diff
@@ -1,4 +1,4 @@
 class ApplicationMailer < ActionMailer::Base
-  default from: 'from@example.com'
+  default from: 'postmaster@sandboxd23676cf8ca74d14b04278ed1d309c95.mailgun.org'
```

Comment:
This variable should be in your env file

Reply:
Now stored in encrypted credentials